### DISCUSSION TITLE
Welcome to OpenSquare off-chain voting Discussions!

### DISCUSSION THREAD

**Discussion by wliyongfeng** [opensquare-network/collaboration#812] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by animatecitizen** [comment]
I with 4 Nay Vot of a referenda what it mean Nay vale deposit in my account within 14 days ?
help me to understand .
